core/controllers/reader.py

```diff
@@ -1371,6 +1371,172 @@ def get(self):
         self.render_json(self.values)
 
 
+class TransientCheckpointUrlHandler(base.BaseHandler):
+    """Responsible for redirecting a logged-out learner."""
```

Comment:
This doesn't really handles any kind of redirecting right. Can you please update the docstring with what it actually does?

Reply:
Done.